sdk/storage/azure-storage-files-shares/swagger/README.md

```diff
@@ -79,13 +79,13 @@ directive:
           "name": "ApiVersion",
           "modelAsString": false
           },
-        "enum": ["2022-11-02"],
+        "enum": ["2023-01-03"],
         "description": "The version used for the operations to Azure storage services."
       };
   - from: swagger-document
     where: $.parameters
     transform: >
-      $.ApiVersionParameter.enum[0] = "2022-11-02";
+      $.ApiVersionParameter.enum[0] = "2023-01-03";
```

Comment:
Similar question to the previous one: Why are you overriding the version in the original swagger?

Reply:
Sometimes the version of swagger is not consistent with the actual ApiVersion, for example, the latest version of the blob in [swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json) is 2021-12-02, but the current version required for the feature is 2022-11-02. So we have to specify the version manually.

Reply:
I know that this isn't your team, but IMHO that is slightly scary because the API breaking change tooling heavily depends on the Api Version in the swagger being the actual API Version (specifically, there are swagger validation tests which will attempt to hit the server with the API version in the swagger file, and they get annoyed if the server doesn't exactly reflect the swagger).